openedx/features/course_experience/templates/course_experience/course-message-fragment.html

```diff
@@ -0,0 +1,75 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.conf import settings
+from django.utils.http import urlquote_plus
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.js_utils import js_escaped_string
+from openedx.core.djangolib.markup import HTML
+from django.core.urlresolvers import reverse
+%>
+
+<%
+# set doc language direction
+from django.utils.translation import get_language_bidi
+is_rtl = get_language_bidi()
+%>
+
+<%block name="content">
+% if user_access['is_anonymous']:
+    <div class="course-message grid-manual">
+        % if not is_rtl:
+        <img class="message-author col col-2" src="${static.url(image_src)}"/>
+        % endif
+        <div class="message-content col ${'col-12' if is_rtl else 'col-10'}">
+            ${HTML(_(
```

Comment:
Our best practice is that the strings that are translated do not contain any HTML markup. For a good description, see: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/preventing_xss.html#html-context-in-mako